nodes/nodes/eco-node-health-check-operator.adoc

```diff
@@ -23,6 +23,10 @@ include::modules/eco-node-health-check-operator-installation-web-console.adoc[le
 
 include::modules/eco-node-health-check-operator-installation-cli.adoc[leveloffset=+1]
 
+[id="about-collecting-data-nhc"]
+== About collecting data for the Node Health Check Operator
```

Comment:
Suggestion:
Collecting Node Health Check operator data for failure debugging

Reply:
@anna-savina Done! I updated the title to address your concerns